docs/pages/web/sheet.js

```diff
@@ -261,6 +273,221 @@ Sheet comes in 3 sizes: small (\`sm\`), medium (\`md\`), and large (\`lg\`).
             }
           />
         </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Dismiss confirmation"
+          description={`There are two ways Sheet can be dismissed: internally-controlled and externally-controlled dismiss actions.
+
+The three internally-controlled or component-controlled dismiss actions are:
+- when the \`ESC\` key is pressed
+- when the backdrop is clicked
+- when the dismiss IconButtons are clicked
+
+The externally-controlled dismiss actions require implementing the callback \`onDismissStart\`. See the [animation variant](#Animation) to learn more.
```

Comment:
Should we list what the three externally-controlled dismiss methods are here?

Reply:
DONE